internal/k8s/validation.go

```diff
@@ -227,6 +231,20 @@ var (
 			validateRequiredAnnotation,
 			validateBoolAnnotation,
 		},
+		appProtectDosEnableAnnotation: {
```

Comment:
There are more Dos annotations:
- appprotectdos.f5.com/app-protect-dos-policy
- appprotectdos.f5.com/app-protect-dos-security-log
-  appprotectdos.f5.com/app-protect-dos-monitor
- appprotectdos.f5.com/app-protect-dos-name

Could you add validation for them? Both that they (1) require Dos and also  (2) validating their values? Note that validating the values here will make validation in `getAppProtectDosLogConfAndDst` unnecessary, because an Ingress with invalid annotations values will not be processed.
	
Note that In AppProtect annotations we have a validation gap which we will address separately later.

Reply:
Done, add validation to all new app protect dos annotation.
Remove the validation from the `getAppProtectDosLogConfAndDst`